.github/workflows/pull_request.yml

```diff
@@ -171,7 +171,7 @@ jobs:
       - name: Finish label
         run: |
           cd "$GITHUB_WORKSPACE/tests/ci"
-          python3 finish_check.py
+          python3 finish_check.py ${{ contains(needs.*.result, 'failure') && '--pipeline-failure' || '' }}
```

Comment:
We discussed this in Slack.

The flag `--pipeline-failure` could work or do nothing based on the `pr_info.event_type`.

I'd like to see the flag working w/o conditions in the script. So set it in the yml based on the event type.

The other way would work too. Parsing the `needs_data` and `pr_info.event_type` in the script. But then the flag is meaningless.